### PR TITLE
reminder history updated: added session message

### PR DESCRIPTION
**Pull request by Hopekodi**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: resources/views/backend/transaction/show.blade.php [hunk 1]
```diff
@@ -8,6 +8,15 @@
     <div class="container-fluid">
         <div class="row-justify-content-center">
 
+            @if(Session::has('message'))
```
Comment (Jeremiahiro) [inline resources/views/backend/transaction/show.blade.php:11]: there is an alert in partials that you could just include

File: resources/views/backend/transaction/show.blade.php [hunk 2]
```diff
@@ -363,6 +372,38 @@ class="mr-2 text-uppercase">
...
+
+                                <input type="hidden" name="message" value="{{old('transaction_id', $transaction->_id)}}">
+
+                                <div class="form-group">
+                                            {{-- <textarea class="form-control"
+                                                        id="reminderMessage" placeholder="Message"></textarea> --}}
+
+                                    <label>Message</label>
+
+                                    <input type="text" name="message" value="{{old('message', $Debts->message)}}">
```
Comment (Jeremiahiro) [inline resources/views/backend/transaction/show.blade.php:394]:
```
<input type="text" class="@error('message') is-invalid @enderror" name="message" value="{{ old('message, $Debts->message') }}">
@error('message')
<span class="invalid-feedback" role="alert">
<strong>{{ $message }}</strong>
</span>
@enderror
```

File: resources/views/backend/transaction/show.blade.php (hunk 1 above)
Comment (kofimokome) [reply resources/views/backend/transaction/show.blade.php:11]: @Hopekodi you did not still add the partials?

Comment (Hopekodi) [reply resources/views/backend/transaction/show.blade.php:11]: done

**Pull request merged**